src/commands/project/init.test.ts

```diff
@@ -14,54 +14,61 @@ afterAll(() => {
+    const randomTmpDir = path.join(os.tmpdir(), uuid());
+    fs.mkdirSync(randomTmpDir);
 
-  // init
-  await initialize(randomTmpDir);
+    // init
+    await initialize(randomTmpDir);
 
-  // Ensure all necessary are created
-  const filepaths = [
-    "bedrock.yaml",
-    "maintainers.yaml",
-    "azure-pipelines.yaml"
-  ].map(filename => path.join(randomTmpDir, filename));
+    // bedrock.yaml, maintainers.yaml, and azure-pipelines.yaml should be in a the root for a 'standard' project
+    const filepaths = [
+      "bedrock.yaml",
+      "maintainers.yaml",
+      "azure-pipelines.yaml"
+    ].map(filename => path.join(randomTmpDir, filename));
 
-  for (const filepath of filepaths) {
-    expect(fs.existsSync(filepath)).toBe(true);
-  }
+    for (const filepath of filepaths) {
+      expect(fs.existsSync(filepath)).toBe(true);
+    }
+  });
 });
 
-test("bedrock.yaml, maintainers.yaml, and azure-pipelines.yaml gets generated for all package directories in a mono-repo", async () => {
-  const randomTmpDir = path.join(os.tmpdir(), uuid());
-  fs.mkdirSync(randomTmpDir);
+describe("Initializing a blank mono-repo", () => {
+  test("bedrock.yaml and maintainers.yaml get generated in the project root and azure-pipelines.yaml gets generated in all package directories in a mono-repo", async () => {
+    const randomTmpDir = path.join(os.tmpdir(), uuid());
+    fs.mkdirSync(randomTmpDir);
 
-  // Create 3 empty stub project
-  const randomPackagesDir = uuid();
-  const randomSubProjectDirs = Array.from({ length: 3 }, (_, i) => {
-    const projectDir = path.join(randomTmpDir, randomPackagesDir, i.toString());
-    expect(shell.mkdir("-p", projectDir).code).toBe(0);
-    return projectDir;
-  });
+    // Create som empty service directories
+    const randomPackagesDir = uuid();
+    const randomSubProjectDirs = Array.from({ length: 3 }, (_, i) => {
+      const projectDir = path.join(
+        randomTmpDir,
+        randomPackagesDir,
+        i.toString()
+      );
+      expect(shell.mkdir("-p", projectDir).code).toBe(0);
+      return projectDir;
+    });
 
-  // Initialize the monorepo
-  await initialize(randomTmpDir, {
-    monoRepo: true,
-    packagesDir: randomPackagesDir
-  });
+    // Initialize the monorepo
+    await initialize(randomTmpDir, {
+      monoRepo: true,
+      packagesDir: randomPackagesDir
+    });
 
-  for (const subProjectDir of randomSubProjectDirs) {
-    // Ensure all sub-projects have the necessary files
-    const filepaths = [
-      "bedrock.yaml",
-      "maintainers.yaml",
-      "azure-pipelines.yaml"
-    ].map(filename => path.join(subProjectDir, filename));
+    // root should have bedrock.yaml and maintainers.yaml
+    for (const file of ["bedrock.yaml", "maintainers.yaml"]) {
+      const filepath = path.join(randomTmpDir, file);
+      expect(fs.existsSync(filepath)).toBe(true);
+    }
 
-    for (const filepath of filepaths) {
+    // All package directories should have an azure-pipelines.yaml
```

Comment:
should we assert that the bedrock.yaml is not created in the subdirs?

Reply:
The way the test is currently laid out, there aren't any subdirs in the project files.

Scaffolding for the test is essentially:

```sh
mkdir -p /<host-tmp-dir>/<random-number-for-project-dir>/packages/
mkdir -p /<host-tmp-dir>/<random-number-for-project-dir>/packages/1
mkdir -p /<host-tmp-dir>/<random-number-for-project-dir>/packages/2
mkdir -p /<host-tmp-dir>/<random-number-for-project-dir>/packages/3
spk project init --mono-repo --packages-dir 
```

Thinking we should add other directories in the dirs?